docs/index.md

```diff
@@ -29,7 +29,41 @@ Each linter rule has its own [rule documentation][], and rules can be
 
 ## Installation
 
-Lorem ipsum dolor set amet...
+To install `api-linter`, use `go get`:
+
+```golang
```

Comment:
This is a shell command, right? So `sh` should be the language?

Reply:
Done. Thanks!